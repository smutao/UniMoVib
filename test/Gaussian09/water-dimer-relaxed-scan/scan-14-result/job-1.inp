a test job

 $contrl
   qcprog="gaussian"
   ifgsva=.true.
 $end

 $gsva
   subsystem="1,2,3" 
 $end

 $qcdata
   fchk="dimer-scan-14-s12.fchk"
 $end
