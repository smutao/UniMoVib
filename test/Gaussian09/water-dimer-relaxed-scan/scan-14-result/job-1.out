 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.4.1, May 02, 2021
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      Gaussian
 FCHK file:       dimer-scan-14-s12.fchk


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    O       8           -3.52349004    0.00216989    0.00279685           15.99491460
     2    H       1           -4.10712401   -0.56660912    0.50572485            1.00782504
     3    H       1           -4.10298405    0.55340687   -0.52389615            1.00782504
     4    O       8            3.75622794   -0.00440188   -0.00433715           15.99491460
     5    H       1            3.21055192    0.54152110    0.56259485            1.00782504
     6    H       1            3.13765295   -0.51046290   -0.53210115            1.00782504
 ------------------------------------------------------------------------------------------


 <<< POINT GROUP SYMMETRY >>>
 Electronic Wavefunctions      :  C1  
 Nuclear & Total Wavefunctions :  C1  


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm^-1), reduced masses (AMU), force constants (mDyn/A), IR intensities (km/mol)

                                                   1                                 2                                 3
          Irreps                                   A                                 A                                 A
     Frequencies                            -10.3234                           24.6747                           36.6942
  Reduced masses                              6.0007                            1.0363                            1.0314
 Force constants                             -0.0004                            0.0004                            0.0008
  IR intensities                              0.0115                           19.3043                           65.2122
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.40812  -0.00022  -0.00026      -0.00249   0.00349   0.02073       0.00028  -0.00185   0.03401
           2   1         0.40826  -0.00100  -0.00094      -0.10770  -0.16164  -0.28815      -0.23354  -0.24128  -0.50819
           3   1         0.40803  -0.00214  -0.00220       0.11442  -0.00759  -0.11956       0.24327   0.29153   0.07365
           4   8        -0.40814  -0.00055  -0.00066      -0.00027  -0.02050  -0.03216       0.00009   0.02033  -0.00103
           5   1        -0.39155   0.00638   0.00869       0.54310   0.28685   0.19461      -0.22186   0.12513  -0.31539
           6   1        -0.42437   0.00903   0.00908      -0.50595   0.15237   0.39444       0.20620  -0.46869   0.22647

                                                   4                                 5                                 6
          Irreps                                   A                                 A                                 A
     Frequencies                             48.4015                           70.2855                          115.7026
  Reduced masses                              1.0536                            1.1348                            1.0994
 Force constants                              0.0015                            0.0033                            0.0087
  IR intensities                             23.9824                          352.0284                          419.2706
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8        -0.00027  -0.02598   0.03997       0.00182  -0.07215  -0.05578      -0.00007   0.02026  -0.01505
           2   1        -0.46324   0.36230  -0.05826      -0.17014   0.44597   0.33065       0.18538  -0.08525   0.08081
           3   1         0.46923  -0.02089  -0.47144       0.13747   0.57781   0.47527      -0.18513  -0.11321   0.04886
           4   8         0.00014   0.02218  -0.01712      -0.00023   0.00754  -0.00982       0.00019   0.05349  -0.05113
           5   1        -0.05471  -0.33340   0.27249       0.11784   0.07248   0.04109      -0.00760  -0.48575   0.46056
           6   1         0.05075   0.05241  -0.10537      -0.11032  -0.07085   0.19414       0.00551  -0.48620   0.46011

                                                   7                                 8                                 9
          Irreps                                   A                                 A                                 A
     Frequencies                           1611.7341                         1613.3287                         3898.4871
  Reduced masses                              1.0827                            1.0827                            1.0452
 Force constants                              1.6571                            1.6604                            9.3594
  IR intensities                             85.0603                           54.3475                           18.0230
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.07019   0.00106   0.00142       0.00812   0.00012   0.00016       0.00951   0.00016   0.00018
           2   1        -0.55583   0.30449  -0.29875      -0.06387   0.03503  -0.03444      -0.07595  -0.08339   0.07407
           3   1        -0.55823  -0.32126   0.27618      -0.06415  -0.03700   0.03180      -0.07506   0.08082  -0.07690
           4   8         0.00807  -0.00027  -0.00027      -0.07016   0.00238   0.00234       0.04899  -0.00142  -0.00138
           5   1        -0.06639  -0.03177  -0.03301       0.57709   0.27546   0.28771      -0.35895   0.40831   0.42415
           6   1        -0.06161   0.03611   0.03729       0.53551  -0.31316  -0.32479      -0.41853  -0.38577  -0.40221

                                                  10                                11                                12
          Irreps                                   A                                 A                                 A
     Frequencies                           3898.6705                         4006.0948                         4007.3229
  Reduced masses                              1.0452                            1.0824                            1.0825
 Force constants                              9.3597                           10.2349                           10.2423
  IR intensities                              8.1953                           62.5921                           63.5125
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8        -0.04897  -0.00083  -0.00091      -0.00000  -0.00004   0.00005       0.00010   0.05198  -0.04778
           2   1         0.39092   0.42920  -0.38119       0.00039   0.00033  -0.00040      -0.42916  -0.41813   0.36967
           3   1         0.38631  -0.41602   0.39571      -0.00034   0.00027  -0.00036       0.42762  -0.40679   0.38861
           4   8         0.00952  -0.00028  -0.00027       0.00311   0.04883   0.05082       0.00000   0.00004   0.00004
           5   1        -0.06981   0.07932   0.08242       0.40375  -0.40444  -0.42005       0.00034  -0.00037  -0.00034
           6   1        -0.08133  -0.07489  -0.07811      -0.45304  -0.37060  -0.38652      -0.00039  -0.00034  -0.00031

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                   A                                 A                                 A
     Frequencies                             -0.0146                            0.0439                           -0.0083
  Reduced masses                              6.0035                            6.0035                            6.0035
 Force constants                             -0.0000                            0.0000                           -0.0000
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           2   1         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           3   1         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           4   8         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           5   1         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825
           6   1         0.40825   0.00000   0.00000       0.00000   0.40825   0.00000       0.00000   0.00000   0.40825

                                                   4                                 5                                 6
          Irreps                                   A                                 A                                 A
     Frequencies                             35.5656                            5.1952                            5.6121
  Reduced masses                              1.0079                            5.8705                            5.8675
 Force constants                              0.0008                            0.0001                            0.0001
  IR intensities                              0.0418                            1.6906                            1.6905
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8        -0.00000  -0.00104   0.00081       0.00035  -0.00085   0.39551      -0.00030  -0.39538  -0.00085
           2   1        -0.00029  -0.33168  -0.37346       0.05613  -0.00101   0.46005       0.06249  -0.45996  -0.00102
           3   1         0.00030   0.34556   0.36324      -0.05806  -0.00096   0.45965      -0.06114  -0.45945  -0.00096
           4   8        -0.00000   0.00157  -0.00160      -0.00044   0.00088  -0.40992       0.00043   0.40979   0.00088
           5   1         0.00001  -0.37117   0.35734       0.06215   0.00073  -0.34952      -0.06008   0.34941   0.00078
           6   1        -0.00001   0.34889  -0.33462      -0.05871   0.00076  -0.34150       0.05653   0.34140   0.00071


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Molecular mass            :        36.021129    AMU
 Electronic total energy   :         0.000000    Hartree
 Scaling factor of Freq.   :         1.000000
 Tolerance of scaling      :         0.000000    cm^-1
 Rotational symmetry number:         1
 The C1   point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                  8.319538         1712.590806         1712.775154
           X                         1.000000            0.000401            0.000432
           Y                        -0.000400            0.999998           -0.002150
           Z                        -0.000433            0.002150            0.999998

 Rotational temperatures            10.410892            0.050575            0.050569    Kelvin
 Rot. constants A, B, C              7.235941            0.035151            0.035147    cm^-1
                                   216.928049            1.053808            1.053694    GHz


 #   1    Temperature =       298.15000 Kelvin         Pressure =         1.00000 Atm
 ====================================================================================

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.044040           27.635618
 Thermal correction to Energy               :            0.050967           31.982466
 Thermal correction to Enthalpy             :            0.051911           32.574952
 Thermal correction to Gibbs Free Energy    :            0.011412            7.161080
 ====================================================================================


 -- Generalized Subsystem Vibrational Analysis (GSVA) --

 Subsystem has     3  atoms:
    1    2    3


 *****************************************************************
 ***  Properties of Intrinsic Fragmental Vibrations from GSVA  ***
 *****************************************************************

 Results of vibrations:
 Normal frequencies (cm^-1), reduced masses (AMU), force constants (mDyn/A)

                                                   1                                 2                                 3
     Frequencies                           1610.9503                         3898.5534                         4006.7794
  Reduced masses                              1.0827                            1.0452                            1.0825
 Force constants                              1.6554                            9.3592                           10.2396
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.07065   0.00106   0.00145       0.04990   0.00089   0.00090       0.00007   0.05198  -0.04779
           2   1        -0.55941   0.30652  -0.30101      -0.39864  -0.43751   0.38858      -0.42889  -0.41771   0.36934
           3   1        -0.56188  -0.32330   0.27798      -0.39324   0.42343  -0.40280       0.42779  -0.40720   0.38906


 UniMoVib job terminated correctly, Sun May 16 12:56:36 2021
